### DISCUSSION TITLE
C10 & C12 Capacitor substitutes

### DISCUSSION THREAD

**Discussion by GlennFR** [picocomputer/community#114] [Q&A]
Hi all,  If you try to use the standard BOM to order the passive components you will find that C10 & C12 will be flagged as OBSOLETE.
Don't worry : there are substitutes.  You can try 
1) RDEC71E476MWK1H03B (Murata)
2) FG23X7R1C476MRT00 (TDK)
3) KTD250B476M76A0T00 (Chemi-Con)
They are all available at Mouser.

**Comment by ghhann** [comment]
Any reason you cannot substitute a 47uf 25v electrolytic capacitor?

**Reply by GlennFR** [reply]
None, really - apart from ugliness and long-term reliability...

**Comment by ghhann** [comment]
Thanks I like to use up what I have “in stock” but I found some smt 47uf capacitors I could use as wellThank you On Sep 20, 2024, at 2:52 AM, GlennFR ***@***.***> wrote:﻿
None, really - apart from ugliness and long-term reliability...
